src/main/java/software/amazon/smithy/lsp/ext/FileCachingCollector.java

```diff
@@ -274,7 +274,9 @@ private Boolean shapeWasDefinedInline(OperationShape operation, Shape shape, Mod
         }
         int boundary = Math.max(priorShapeLine, operation.getSourceLocation().getLine());
         while (shapeStartLine >= boundary) {
-            String line = modelFile.lines.get(shapeStartLine);
+            String line = modelFile.lines.get(shapeStartLine - 1);
```

Comment:
I'm not 100% sure this is the right thing to do, but I checked around to confirm, and it seems like there's usually a `-1` happening when we look up lines in other places. I do think this is the right behavior.